lib/run.go

```diff
@@ -131,9 +131,16 @@ func (a *ACBuild) Run(cmd []string, insecure bool) (err error) {
 	}
 
 	if man.App != nil {
+		setPath := false
 		for _, evar := range man.App.Environment {
+			if evar.Name == "PATH" {
```

Comment:
Strictly speaking you don't even need to bother with this right now, the spec mandates it's always this value: https://github.com/appc/spec/blame/master/spec/ace.md#L92